pkgs/data/fonts/lao/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation {
```

This was missed from the previous suggestion.
Needed to access the `version` attr.

```suggestion
stdenv.mkDerivation rec {
```